aci/resource_aci_cloudctxprofile.go

```diff
@@ -241,14 +266,7 @@ func resourceAciCloudContextProfileCreate(ctx context.Context, d *schema.Resourc
 		}
 
 	}
-	if relationTocloudRsCtxProfileToRegion, ok := d.GetOk("relation_cloud_rs_ctx_profile_to_region"); ok {
-		relationParam := relationTocloudRsCtxProfileToRegion.(string)
-		err = aciClient.CreateRelationcloudRsCtxProfileToRegionFromCloudContextProfile(cloudCtxProfile.DistinguishedName, relationParam)
-		if err != nil {
-			return diag.FromErr(err)
-		}
 
-	}
```

Comment:
Don't we still need this part but feed the data from region directly?

Reply:
Yes, we are directly pushing the region instead of using the relation_cloud_rs_ctx_profile_to_region attribute.